src/graphics/program-lib/programs/standard.js

```diff
@@ -326,6 +326,14 @@ const standard = {
         return code;
     },
 
+    _decodeFunc: function (textureFormat) {
+        return {
```

Comment:
this could be constructed one time instead of inside each function call. Or perhaps a switch should be used.

Reply:
done